app/main/views/users.py

```diff
@@ -90,19 +103,23 @@ def download_users(framework_slug):
 
 
 @main.route('/users/download/buyers', methods=['GET'])
-@role_required('admin-framework-manager')
+@role_required('admin-framework-manager', 'admin')
 def download_buyers():
 
     user_attributes = (
         "emailAddress",
         "name",
     )
+
+    if current_user.role == 'admin':
+        user_attributes = user_attributes + ('userResearchOptedIn',)
+
     users = data_api_client.find_users_iter(role="buyer")
 
     rows_iter = chain(
         (
             # header row
-            ("email address", "name"),
+            ("email address", "name", "user research opted in" if current_user.role == 'admin' else None),
```

Comment:
So I didn't realise this but dropping in `None` here adds an empty column to the csv header apparently 😒